### PR TITLE
Display current server time in /modlog

### PR DESCRIPTION
**Pull request by Hidden50**
http://i.imgur.com/X0KehCO.png

This gives a point of reference, to know how the modlog's timestamps
translate to your own timezone.

### CODE REVIEW

**Comment by Zarel** [comment]
May not be a good idea to pullreq two things that different in the same pullreq. Can you split these out?

Also have you done the license in the past?

**Comment by Hidden50** [comment]
I'll rebase this pr to only contain the modlog change, and open another one for /whoare.

Here's my license post (this was really hard to dig up, neither github nor google could find it even when I remembered the exact phrasing): https://github.com/Zarel/Pokemon-Showdown/pull/2396#issuecomment-175923508

(I'm Orda-Y, btw.)

**Comment by Zarel** [comment]
You haven't removed the `/whoare` change from this PR yet.

**Comment by Hidden50** [comment]
Looks like I had synched again by accident. Should be good, now.

File: commands.js [hunk 1]
```diff
@@ -2081,15 +2081,15 @@ exports.commands = {
 				if (!stdout) {
 					connection.popup("The modlog is empty. (Weird.)");
 				} else {
-					connection.popup("|wide||html|<p>The last " + lines + " lines of the Moderator Log of " + roomNames + ":</p>" + stdout);
+					connection.popup("|wide||html|<p>The last " + lines + " lines of the Moderator Log of " + roomNames + ".</p><p><small>[" + Tools.toTimeStamp(new Date(), {hour12: true}) + "]</small> Current server time.</p>" + stdout);
```
Comment (Zarel) [inline commands.js]: I'm gonna suggest "← current server time"

The capitalization and period are very out of place for a sentence fragment.

Comment (Hidden50) [inline commands.js]: Done. http://imgur.com/EQLpTWU

Also made the <small> tag cover the whole line, which makes it seem more like it's part of the header and less like a modlog entry.

**Comment by Slayer95** [comment]
Looks weird. What do you think about displaying the current time on the top-right corner instead?

**Comment by Hidden50** [comment]
Here's what Slayer95's proposal would look like: http://i.imgur.com/adDj2Fu.png

**Comment by Zarel** [comment]
Doesn't look bad I guess...

**Comment by Zarel** [comment]
maybe add brackets and omit "Current server time:" - maybe just "now:" inline

**Comment by Lord-Haji** [comment]
feelscool

**Comment by Zarel** [comment]
Going to merge this, if anyone has a problem with it they can pullreq/commit a redesign

**Pull request merged**